src/components/NavigationSection.svelte

```diff
@@ -77,7 +76,7 @@
       }
     },
     computed: {
-      slug: label => slug(label)
+      hash: label => Math.abs([].reduce.call(label, (hash, char) => char.charCodeAt(0) + (hash << 5) + (hash << 16) - hash, 0))
```

Comment:
Can you explain me that or if you have a source just for my understanding here. Thanks 👍